src/content/docs/usage/builders.mdx

```diff
@@ -58,4 +58,26 @@ The following options are accepted when using this Builder:
 
 ### None
 Daytona can build a Project using the default base image.
-Unless otherwise configured using `daytona server configure`, the default base image is set to `daytonaio/workspace-project`.
\ No newline at end of file
+Unless otherwise configured using `daytona server configure`, the default base image is set to `daytonaio/workspace-project`.
+
+## Setting a Custom Build Registry
```

Comment:
The steps outlined below are only the first part of configuring the Custom Build Registry. After this, the user needs to run `daytona server configure` and choose that registry from the list in the `Build Registry` filed.

Reply:
Fixed in [42c08f1](https://github.com/daytonaio/docs/pull/62/commits/42c08f1ee61529ba68324ee14f004f9025fe7799)